.github/workflows/release.yml

```diff
@@ -75,7 +79,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: NuGet authentication for GitHub
+      - name: Authenticate to GitHub registry
```

Comment:
Is the authentication step necessary? I think it's possible to inline all required parameters in a single push command.

Reply:
According to GitHub's documentation, the 2-step process is required. https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry#publishing-a-package